db/migrate/20200205214553_add_pbs_id_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddPbsIdToUser < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :pbs_id, :integer
```

Comment:
@diegosteiner is it possible to get the users `pbs_id`? Otherwise i cannot tell, which are his camps.